src/test/tests/connectionManager.test.ts

```diff
@@ -87,6 +90,11 @@ describe('Connection Manager Tests', () => {
                 expectedXrayUrl: 'https://httpbin.org/status/404/different-xray-url/'
             }
         ].forEach(async testCase => {
+            nock('https://httpbin.org/anything')
+                .get('/xray/api/v1/system/ping')
+                .reply(200, 'RESPONSE')
+                .get('/artifactory/api/system/ping')
+                .reply(200, 'RESPONSE');
```

Comment:
I'm not sure I understand the logic behind this. It is intended to use httpbin with anything.
```suggestion
```

Reply:
Without this, the test fails.

Reply:
They don't fail in my environment and in CI. This code doesn't make sense. We can talk about it after the release.

Reply:
Removed.